testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -62,7 +74,7 @@ impl EventObserver {
             let val = json!({
                 "txid": format!("0x{}", tx.txid()),
                 "tx_index": tx_index,
-                "success": success,
+                "status": success,
```

Comment:
@zone117x is this change breaking compatibility with the sidecar currently deployed?